### PR TITLE
runtimevar: rework URL comments for consistency, and fix to use env vars

### PR DESCRIPTION
**Pull request by vangent**
This is similar to #1533, but for `runtimevar`.

In this one, `etcdvar` is updated to use an environment variable.

I also removed the "wait" URL parameters; we can always add them later if requested.

Updates #1174.

### CODE REVIEW

File: runtimevar/runtimevar.go [hunk 1]
```diff
@@ -482,14 +482,20 @@ func bytesDecode(b []byte, obj interface{}) error {
 // DecoderByName returns a *Decoder based on decoderName.
 // It is intended to be used by VariableURLOpeners in driver packages.
 // Supported values include:
-//   - (empty string), "bytes": Returns the default, BytesDecoder;
-//       Snapshot.Valuewill be of type []byte.
+//   - empty string: Returns the URLEncoder default, or BytesDecoder
```
Comment (shantuo) [inline runtimevar/runtimevar.go]: What is a `URLEncoder`? How to set the default?

File: runtimevar/blobvar/example_test.go [hunk 2]
```diff
@@ -69,11 +69,11 @@ func Example_openVariable() {
 	// OpenVariable creates a *runtimevar.Variable from a URL.
 	// This example watches a variable based on a file-based blob.Bucket with JSON.
 	ctx := context.Background()
-	v, err := runtimevar.OpenVariable(ctx, "blob://myvar.json?bucket=file:///mypath&decoder=json")
+	v, err := runtimevar.OpenVariable(ctx, "blob://myvar.json?bucket=file%3A%2F%2F%2Fmypath&decoder=json")
```
[the remark this answers is not in the document]
Comment (vangent) [reply runtimevar/blobvar/example_test.go]: Done.

File: runtimevar/runtimevar.go (hunk 1 above)
Comment (vangent) [reply runtimevar/runtimevar.go]: Sorry, should have been `URLOpener.Decoder`. I clarified the comment.

**Comment by vangent** [comment]
PTAL

**Pull request merged**